[2023-07-20T14:19:04.809+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my-first-dag.extract scheduled__2023-07-19T00:00:00+00:00 [queued]>
[2023-07-20T14:19:04.815+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my-first-dag.extract scheduled__2023-07-19T00:00:00+00:00 [queued]>
[2023-07-20T14:19:04.816+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-07-20T14:19:04.825+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): extract> on 2023-07-19 00:00:00+00:00
[2023-07-20T14:19:04.829+0000] {standard_task_runner.py:57} INFO - Started process 160 to run task
[2023-07-20T14:19:04.832+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my-first-dag', 'extract', 'scheduled__2023-07-19T00:00:00+00:00', '--job-id', '1113', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpkwzeb7kc']
[2023-07-20T14:19:04.835+0000] {standard_task_runner.py:85} INFO - Job 1113: Subtask extract
[2023-07-20T14:19:04.877+0000] {task_command.py:410} INFO - Running <TaskInstance: my-first-dag.extract scheduled__2023-07-19T00:00:00+00:00 [running]> on host 70d871733d27
[2023-07-20T14:19:04.933+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ramesh@somemail.com' AIRFLOW_CTX_DAG_OWNER='Ramesh Sannareddy' AIRFLOW_CTX_DAG_ID='my-first-dag' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-07-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-19T00:00:00+00:00'
[2023-07-20T14:19:04.935+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-20T14:19:04.936+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cut -d":" -f1,3,6 /etc/passwd > /home/project/***/dags/extracted-data.txt']
[2023-07-20T14:19:04.945+0000] {subprocess.py:86} INFO - Output:
[2023-07-20T14:19:04.948+0000] {subprocess.py:93} INFO - /bin/bash: line 1: /home/project/***/dags/extracted-data.txt: No such file or directory
[2023-07-20T14:19:04.949+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-07-20T14:19:04.959+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-07-20T14:19:04.962+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=my-first-dag, task_id=extract, execution_date=20230719T000000, start_date=20230720T141904, end_date=20230720T141904
[2023-07-20T14:19:04.970+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 1113 for task extract (Bash command failed. The command returned a non-zero exit code 1.; 160)
[2023-07-20T14:19:05.006+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-20T14:19:05.025+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
